package.json

```diff
@@ -69,7 +68,7 @@
     "lint": "eslint -c .eslintrc.js --ext .ts ./src/ ./test/",
     "lint:fix": "eslint --fix -c .eslintrc.js --ext .ts ./src/ ./test/",
     "build": "rm -rf lib && tsc",
-    "prepublishOnly": "npm run build && npm run test && npm run tslint"
+    "prepublishOnly": "npm run build && npm run test && npm run lint"
```

Comment:
Question, is the eslint based task also stopping the `publish` action?

Reply:
yes - by intention - simply replacing tslint